pkg/server/application_api/application_api_test: TestHotRanges2ResponseWithViewActivityOptions failed

pkg/server/application_api/application_api_test.TestHotRanges2ResponseWithViewActivityOptions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/10982481?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/10982481?buildTab=artifacts#/) on master @ [bb9b3fd5f0c03f795e2824062b98a49b6999b20a](https://github.com/cockroachdb/cockroach/commits/bb9b3fd5f0c03f795e2824062b98a49b6999b20a):

Fatal error:

```
panic: test timed out after 4m55s

```
Stack: 

```
goroutine 423926 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x88
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestHotRanges2ResponseWithViewActivityOptions
    test_log_scope.go:167: test logs captured to: /artifacts/tmp/_tmp/bdcf2ab810bc38279e9dce2da8e9dabd/logTestHotRanges2ResponseWithViewActivityOptions2661847168
    test_log_scope.go:81: use -show-logs to present logs inline
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/unowned
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRanges2ResponseWithViewActivityOptions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-29952

--- a/pkg/server/application_api/BUILD.bazel
+++ b/pkg/server/application_api/BUILD.bazel
@@ -9,6 +9,7 @@ go_library(
 
 go_test(
     name = "application_api_test",
+    size = "large",
     srcs = [
         "activity_test.go",
         "config_test.go",
@@ -30,7 +31,7 @@ go_test(
         "util_test.go",
         "zcfg_test.go",
     ],
-    args = ["-test.timeout=295s"],
+    args = ["-test.timeout=895s"],
     deps = [
         "//pkg/base",
         "//pkg/ccl",

